src/olympia/ratings/tests/test_views.py

```diff
@@ -2274,12 +2332,36 @@ def test_no_throttle(self):
         )
         url_b = reverse_ns(self.flag_url_name, kwargs={'pk': rating_b.pk})
 
-        response = self.client.post(self.url, data={'flag': 'review_flag_reason_spam'})
-        assert response.status_code == 202
-        response = self.client.post(url_b, data={'flag': 'review_flag_reason_spam'})
-        assert response.status_code == 202
-        # Both should have been flagged.
-        assert RatingFlag.objects.count() == 2
+        with freeze_time('2021-07-23') as frozen_time:
+            response = self.client.post(
+                self.url, data={'flag': 'review_flag_reason_spam'}
+            )
+            assert response.status_code == 202
+            response = self.client.post(url_b, data={'flag': 'review_flag_reason_spam'})
+            assert response.status_code == 202
+            # Both should have been flagged.
+            assert RatingFlag.objects.count() == 2
+
+            for x in range(0, 18):
+                # You can keep flagging up to 20 a day.
+                response = self.client.post(
+                    url_b, data={'flag': 'review_flag_reason_spam'}
+                )
+                assert response.status_code == 202
+
+            # Then it becomes too much
```

Comment:
The fact that you sometimes use full stops but not all the time triggered my OCD. Can we have one here (and above)? 😺